AEPIdentity/Tests/IdentityFunctionalTests.swift

```diff
@@ -51,6 +73,61 @@ class IdentityFunctionalTests: XCTestCase {
         XCTAssertNotNil(sharedState?[IdentityConstants.EventDataKeys.VISITOR_ID_ECID])
     }
 
+    /// Tests that a sync event dispatches a shared state update
+    func testSyncIdentifiersForceSyncEventInstallScenario() {
+        // setup
+        setDefaultSuccessNetworkResponse()
+        let configSharedState = [IdentityConstants.Configuration.GLOBAL_CONFIG_PRIVACY: PrivacyStatus.optedIn.rawValue, IdentityConstants.Configuration.EXPERIENCE_CLOUD_ORGID: "test-org-id"]
+        let data = [IdentityConstants.EventDataKeys.IS_SYNC_EVENT: true]
+        let event = Event(name: "Sync Event", type: EventType.identity, source: EventSource.requestIdentity, data: data)
+        mockRuntime.simulateSharedState(extensionName: "com.adobe.module.configuration", event: event, data: (configSharedState, .set))
+        waitForProcessing()
+
+        // test
+        XCTAssertTrue(identity.readyForEvent(event)) //// since first event will be processed by readyForEvent which calls forceSync
+        waitForProcessing()
```

Comment:
This wait can be flaky as the processing time may defer on each testing machine, instead we should wait for the network request to execute to do the assertions.
I would recommend moving all these tests inside https://github.com/adobe/aepsdk-core-ios/blob/main/AEPIntegrationTests/IdentityIntegrationTests.swift and using the TestableNetworkService and the RequestResolver for this.